src/backend/renderer.js

```diff
@@ -1803,6 +1807,23 @@ export function attach(
     const isTimedOutSuspense =
       tag === SuspenseComponent && memoizedState !== null;
 
+    let events = null;
+    let node = fiber;
+    while (node !== null) {
+      if (node.tag === EventComponent) {
+        if (events === null) {
+          events = [];
+        }
+        const eventComponentInstance = node.stateNode;
+        const currentFiber = eventComponentInstance.currentFiber;
+        events.push({
+          props: eventComponentInstance.props,
+          displayName: currentFiber.type.displayName,
```

Comment:
Will this always be a (non-empty) string or should we have a fallback (e.g. "Anonymous")?

Also we should probably use `getDisplayNameForFiber(currentFiber)` for consistency here. (That's what we do elsewhere.)

Looks like `getDisplayNameForFiber` doesn't yet handle `EventComponent` types.

Reply:
`displayName` is no longer optional as far as I'm aware, so we should be fine.

Reply:
> Also we should probably use `getDisplayNameForFiber(currentFiber)` for consistency here. (That's what we do elsewhere.)

> Looks like `getDisplayNameForFiber` doesn't yet handle EventComponent types.

Doesn't seem like either fo these changes were made 😁

Reply:
I guess I can just make them while I'm in here.

Reply:
Sorry I didn’t get around to it today. :( thank you for the help!

Reply:
No problem!